### ISSUE TITLE
Description Always Shows ‘more’ Button

### ISSUE THREAD

**Issue by aphronyx** [Aidoku/Aidoku#272]
The description section in manga view can show up to four lines without expanding; however, even if the full description is less than five lines, the description section still shows a ‘more’ button.

In short, **the description section shows a ‘more’ button when it don’t need to be expanded**.

## Screenshots

| Expected | Actual |
| :-: | :-: |
| ![IMG_1189](https://github.com/Aidoku/Aidoku/assets/64679454/3ce52d48-0c1b-401a-a15f-391d8987eea5) | ![IMG_1188](https://github.com/Aidoku/Aidoku/assets/64679454/d4a1086b-33aa-411e-981f-0ef8a883334a) |
| ![IMG_1191](https://github.com/Aidoku/Aidoku/assets/64679454/923a06de-a0cb-4d18-87ed-c49f0d916c8c) | ![IMG_1190](https://github.com/Aidoku/Aidoku/assets/64679454/f19b5344-7e44-49dc-8969-99e52b09803b) |
| ![IMG_1187](https://github.com/Aidoku/Aidoku/assets/64679454/79aa9fcf-21ee-4003-886e-958574edb5ee) | ![IMG_1186](https://github.com/Aidoku/Aidoku/assets/64679454/aea4fc1c-b49d-457f-b864-da786307e9bd) |

## Info

- Aidoku version: 0.6.3 (1)
- iOS version: 16.5.1
- Device model: iPhone SE

**Comment by Skittyblock** [comment]
Yeah I know, I couldn't figure out how to properly detect the actual amount of lines for the text when I made it. I'm sure there's some way of doing it.

**Comment by aphronyx** [comment]
@Skittyblock should this be closed for the new description view?

**Comment by Skittyblock** [comment]
should probably wait until the old manga view is fully gone because technically it's still like this in the library (and i think a couple other places)
